.github/ISSUE_TEMPLATE/docs-request-internal.yaml

```
@@ -1,5 +1,5 @@
 name: "Internal documentation request (Elastic employees) "
-description: Request a documentation change or enhancement.
+description: Request a documentation change or enhancement. For tasks you don't want exposed to the public, create an issue in https://github.com/elastic/security-docs-internal.
```

I don't know if GitHub will automatically shorten this link, like it does in PRs and issues, but this will definitely shorten it:
```suggestion
description: Request a documentation change or enhancement. For tasks you don't want exposed to the public, create an issue in [elastic/security-docs-internal](https://github.com/elastic/security-docs-internal).
```